File: Formacao-Flutter/imcapp/lib/classes/pessoa.dart
class Pessoa {
  late String _nome;
  late double _altura;
  late double _peso;

  void setNome(String nome) {
    _nome = nome;
  }

  String getNome() {
    return _nome;
  }

  void setAltura(double altura) {
    _altura = altura;
  }

  double getAltura() {
    return _altura;
  }

  void setPeso(double peso) {
    _peso = peso;
  }

  double getPeso() {
    return _peso;
  }

  double calcularIMC() {
    return _peso / (_altura * _altura);
  }
}

File: Bootcamp-XP/SistemaHospedagem/Models/Pessoa.cs

namespace SistemaHospedagem.Models
{
    public class Pessoa
    {
        public Pessoa(string nome, string sobrenome)        
        {
            this.Nome = nome;
            this.Sobrenome = sobrenome;
        }

        public string Nome { get; set; }
        public string Sobrenome { get; set; }
    }
}